### ISSUE TITLE
Very change diff returns an error

### ISSUE THREAD

**Issue by elliots** [KanHarI/gpt-commit-summarizer#22]
I got an "Error: OpenAI query too big" testing it with a large xml file with a bunch of changes.

Maybe limit the diff to a reasonable size and include a note that it was trimmed? 

(Very cool by the way)

**Comment by elliots** [comment]
Ah, i see that it is already supposed to throw an error if its too big...

https://github.com/KanHarI/gpt-commit-summarizer/blob/master/src/summarizePr.ts#L46-L48

**Comment by jjl-genemod** [comment]
Yeah it would be nice if there was more fine grained control over when it comments on large prs.

PS: Thank you to the creator of this. It's very very cool!
